testing-plugin/src/main/kotlin/arrow/meta/plugin/testing/CompilationAssertions.kt

```diff
@@ -143,6 +144,16 @@ private fun assertFailsWith(compilationResult: Result, check: (String) -> Boolea
   assertThat(check(compilationResult.messages)).isTrue()
 }
 
+private fun assertQuoteFileMatches(compilationResult: Result, expectedFileName: String, expectedFileSource: Code.Source) {
+  assertCompiles(compilationResult)
+  val actualSource = Paths.get("build", "generated", "source", "kapt", "main", expectedFileName).toFile().readText()
+  val actualSourceWithoutCommands = removeCommands(actualSource)
+  val expectedSourceWithoutCommands = removeCommands(expectedFileSource.text.trimMargin())
+  assertThat(actualSourceWithoutCommands)
+          .`as`("EXPECTED:${expectedFileSource.text.trimMargin()}\nACTUAL:$actualSource\nNOTE: Meta commands are skipped in the comparison")
+          .isEqualToIgnoringWhitespace(expectedSourceWithoutCommands)
+}
+
 private fun assertQuoteOutputMatches(compilationResult: Result, expectedSource: Code.Source): Unit {
```

Comment:
I think that the body of this method could be a call to `assertQuoteFileMatches`.